perceval/backend.py

```diff
@@ -74,14 +74,27 @@ class Backend:
     Classified data filtering and archiving are not compatible to prevent
     data leaks or security issues.
 
+    Each backend can also provides a set of search terms to simplify query
```

Comment:
I would changed this with:

```suggestion
    Each backend might provide a set of search terms to simplify query
```

Reply:
I'm not sure about the use of `might` (which expresses permission) over `can` (which expresses ability/capability) in this sentence.
I'll fix the typo (`provides`)